src/Valit/IValitRules.cs

```diff
@@ -5,16 +5,17 @@ namespace Valit
 {
     public interface IValitRules<TObject> where TObject : class
     {
-        IValitRules<TObject> Ensure<TProperty>(Func<TObject, TProperty> selector, Func<IValitRule<TObject, TProperty>, IValitRule<TObject, TProperty>> ruleFunc);
-        IValitRules<TObject> Ensure<TProperty>(Func<TObject, TProperty> selector, IValitRulesProvider<TProperty> valitRulesProvider) where TProperty : class;
-        IValitRules<TObject> EnsureFor<TProperty>(Func<TObject, IEnumerable<TProperty>> selector, Func<IValitRule<TObject, TProperty>, IValitRule<TObject, TProperty>> ruleFunc);
-        IValitRules<TObject> EnsureFor<TProperty>(Func<TObject, IEnumerable<TProperty>> selector, IValitRulesProvider<TProperty> valitRulesProvider) where TProperty : class;
-        IValitRules<TObject> For(TObject @object);
-        IEnumerable<IValitRule<TObject>> GetAllRules();
-        IEnumerable<IValitRule<TObject>> GetTaggedRules();
-        IEnumerable<IValitRule<TObject>> GetUntaggedRules();
-        IValitResult Validate();
-        IValitResult Validate(params string[] tags);
-        IValitResult Validate(Func<IValitRule<TObject>, bool> predicate);
+         IValitRules<TObject> Ensure<TProperty>(Func<TObject, TProperty> selector, Func<IValitRule<TObject, TProperty>, IValitRule<TObject, TProperty>> ruleFunc);
+         IValitRules<TObject> Ensure<TProperty>(Func<TObject, TProperty> selector, IValitRulesProvider<TProperty> valitRulesProvider) where TProperty : class;
+         IValitRules<TObject> Ensure<TProperty>(Func<TObject, TProperty> selector, Func<IValitRule<TObject, TProperty>, TObject, IValitRule<TObject, TProperty>> rule);
+         IValitRules<TObject> EnsureFor<TProperty>(Func<TObject, IEnumerable<TProperty>> selector, Func<IValitRule<TObject, TProperty>, IValitRule<TObject, TProperty>> ruleFunc);
+         IValitRules<TObject> EnsureFor<TProperty>(Func<TObject, IEnumerable<TProperty>> selector, IValitRulesProvider<TProperty> valitRulesProvider) where TProperty : class;
+         IValitRules<TObject> For(TObject @object);
+         IEnumerable<IValitRule<TObject>> GetAllRules();         
```

Comment:
Here are some extra whitespaces at the end of the line. (also an extra space for every property in this interface)
I don't know why, but I thought the `.editorconfig` would fix these. 

Can I ask which editor you're using? 😄

Reply:
Visual Studio Code on macOS. I will take a look on this.

Reply:
I forgot this, but it seems I have [this extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) installed...